File: uri/1037.cpp
#include <iostream>

int32_t main(void){
	double n;
	std::cin>>n;

	if(n<0 || n>100) std::cout<<"Fora de intervalo"<<std::endl;
	else if(n<=25)std::cout<<"Intervalo [0,25]"<<std::endl; 
	else if(n<=50)std::cout<<"Intervalo (25,50]"<<std::endl; 
	else if(n<=75)std::cout<<"Intervalo (50,75]"<<std::endl; 
	else if(n<=100)std::cout<<"Intervalo (75,100]"<<std::endl; 
}

File: uri/rust/1037.rs
fn get_float() -> f64 {
    let mut buffer = String::new();
    std::io::stdin().read_line(&mut buffer).expect("Failed");
    buffer.trim().parse::<f64>().unwrap()
}

fn main() {
    let n: f64 = get_float();

    if n<0.0 || n>100.0 {
        println!("Fora de intervalo");
    } else if n<=25.0 {
        println!("Intervalo [0,25]");
    } else if n<=50.0 {
        println!("Intervalo (25,50]");
    } else if n<=75.0 {
        println!("Intervalo (50,75]");
    } else if n<=100.0 {
        println!("Intervalo (75,100]");
    }
}
